spec/compute/models/server_spec.rb

```diff
@@ -0,0 +1,196 @@
+require "spec_helper"
+require "fog/compute/models/server"
+
+describe Fog::Compute::Server do
+  before do
+    @server = Fog::Compute::Server.new
+  end
+
+  describe "#sshable?" do
+    before do
+      # I'm not sure why #sshable? depends on a method that's not defined except in implementing classes
```

Comment:
Good question. Maybe we should define it in the base class also, but just have it always return false? That would clear up this confusion/oddity, hopefully. Thoughts?